core/modules/layout/css/layout.admin.css

```diff
@@ -326,3 +326,19 @@
     float: left;
   }
 }
+
+/* Make sure that the machine name wraps under the "Admin label" field. */
+.layout-block-configure-form .form-item-label span.field-suffix {
+  display: block;
```

Comment:
This is to fix this:

![Screen Shot 2019-07-09 at 7 49 50 am](https://user-images.githubusercontent.com/2423362/60848466-0fdd5380-a22a-11e9-815b-f1ca8dfc52f1.png)

...happens only on high-res, wide screens.

Reply:
FTR, we have a dedicated issue to fix this globally: https://github.com/backdrop/backdrop-issues/issues/1349